AppCenterDataStorage/AppCenterDataStorage/MSPaginatedDocuments.h

```diff
@@ -2,13 +2,78 @@
 // Licensed under the MIT License.
 
 #import "MSPage.h"
+#import "MSReadOptions.h"
 #import "MSSerializableDocument.h"
 
 // A (paginated) list of documents from CosmosDB
 @interface MSPaginatedDocuments<T : id <MSSerializableDocument>> : NSObject
 
+// FIXME: these properties should not be part of the public interface.
+
+/**
+ * Current page.
+ */
+@property(nonatomic, strong, readonly) MSPage *currentPage;
```

Comment:
nit: `strong` is default attribute value as opposed to `weak`/`assign`/`copy`, so you don't need declare `strong` here unless you want to emphasize it.